front_end/src/components/searchResultOverview/SortBar.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+// Material UI Imports
+import {
+  withStyles,
+  withWidth,
+  Grid,
+  Typography,
+  Divider,
+  InputLabel,
+  Select,
+  MenuItem,
+  ExpansionPanel,
+  ExpansionPanelSummary,
+  ExpansionPanelDetails
+} from "@material-ui/core";
+import {
+  ArrowDownward,
+  ArrowUpward,
+  Sort,
+  ExpandMore
+} from "@material-ui/icons";
+
+// Component CSS to Javascript styles
+let styles = theme => ({
+  marPadSummary: {
+    paddingLeft: 25,
+    paddingBottom: 5,
+    paddingTop: 5,
+    paddingRight: 25,
+    margin: 0
+  },
+  mar0: {
+    margin: 0
+  },
+  sortButton: { width: "100%" },
+  subtitles: { fontWeight: "bold", color: "#808080" }
+});
+
+let SortBar = props => {
```

Comment:
it is recommended to use **const** instead of **var, let** unless you have to

Reply:
Using 'let' limits the scope of the variable to render() {} while const sets the scope to the entire file.